### DISCUSSION TITLE
Magento paypal buy not available in product page

### DISCUSSION THREAD

**Discussion by arulkailasam** [breezefront/community#310] [1. Help]
In Magento, I'm using the Breeze Evolution Theme. In the Magento_Paypal module, there is a component called paypal-sdk.js that contains the following code:

define(['jquery'], function ($) {
    var dfd = $.Deferred();
});

This code throws an error because the $ object does not contain the Deferred function, although it does have other functions and properties. When I visited the demo website for the Breeze Evolution theme provided by the vendor, the $.Deferred() method is available there.

I'm not sure if I missed any configuration during the theme installation that might be causing this issue. Could you guide me on how to fix this problem?"

**Comment by vovayatsyuk** [comment]
What's the version of your swissup/module-breeze? $.Deferred function was added in 2.14.1.

Please provide URL to your site if Breeze is up to date.

**Reply by arulkailasam** [reply]
Thank you for your time and assistance with my question.
In the Magento root directory, I ran the command composer show and found the following version details:
**swissup/breeze-evolution - 2.1.5**

Could you please provide guidance on the next steps I should take to resolve this issue?

**Reply by vovayatsyuk** [reply]
It's too old version. You need to update Breezefront using `composer update` command.

**Reply by arulkailasam** [reply]
When I ran the command **composer update**, I received the following error message:

"Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Root composer.json requires kommunicate/module-livechat ^2.0, found kommunicate/module-livechat[2.0.1] in the lock file but not in remote repositories, make sure you avoid updating this package to keep the one from the lock file.
  Problem 2
    - Root composer.json requires prodovo/magento2-flatshipping ^2.0, found prodovo/magento2-flatshipping[2.0.4] in the lock file but not in remote repositories, make sure you avoid updating this package to keep the one from the lock file."
